README.md

```diff
@@ -141,6 +141,11 @@ Flow records are uni-directional which means that almost all protocol traffic wi
 |`latency`|The latency experienced for this flow.  For client-side flows (the parent is a listener), this is the latency experienced by the client.  For server-side flows (the parent is a connector), this is the latency of the actual server/pod handling the traffic.  The cross-network latency can be computed by finding the difference between the two latencies.|
 |`trace`|A list (separated by vertical bar characters) of the routers through which this traffic flowed.|
 
+---
+`api/v1alpha1/record?id=<record-id>[&id=<record-id]`
```

Comment:
Just an optimization, but not so important to this stage:

Maybe we can pass parameters by POST so we can directly receive an array of ids instead id1=1&id2=1&id3.=3....&id30=30